tests/rptest/redpanda_cloud_tests/config_profile_verify_test.py

```diff
@@ -60,20 +60,15 @@ def test_config_profile_verify(self):
 
     def _check_rp_config(self):
         confRes = self.redpanda.kubectl.exec(
-            "rpk redpanda admin config print --host 0")
+            'rpk redpanda admin config print --host 0')
         clusterConfig = json.loads(confRes)
-        self.logger.debug(
-            "asserting we got the config for the right cluster: expected rp-{}, actual: {}"
-            .format(self._clusterId, clusterConfig["cluster_id"]))
-        assert clusterConfig['cluster_id'] in (self._clusterId,
-                                               f'rp-{self._clusterId}')
+        assert clusterConfig['cluster_id'] in (
+            self._clusterId, f'rp-{self._clusterId}'
+        ), f'asserting we got the config for the right cluster: expected {self._clusterId} to contain {clusterConfig["cluster_id"]}'
 
         for k, v in self._configProfile["cluster_config"].items():
-            self.logger.debug(
-                "asserting cluster config key {} has expected value: {}  actual: {}"
-                .format(k, v, clusterConfig[k]))
-            if clusterConfig[k] != v and "{}".format(clusterConfig[k]) != v:
-                assert False
+            if clusterConfig[k] != v and f"{clusterConfig[k]}" != v:
```

Comment:
Not suggesting any change right now (existing logic).  Just want to say this is pretty janky.  So we are saying we will allow either basic equality OR stringified equality.

But basic equality implies stringified equality (you would think).

Reply:
Yeah, I don't love this part either. Python does not consider `42` and `'42'` to be equivalent, though, and I'm guessing there's some mismatch between the types of some of the numeric values between sources. Would be good to dig deeper here and see if maybe we could resolve those differences and clean this up.